examples/lines_bars_and_markers/hat_graph.py

```diff
@@ -0,0 +1,59 @@
+"""
+===============================
+Hat Graph with labels
+===============================
+This example shows a how to create a hat graph and how to annotate with labels.
+"""
+import matplotlib
+import numpy as np
+import matplotlib.pyplot as plt
+# initialise labels and a numpy array make sure you have N labels of N number of values in the array
+labels = ['I', 'II','III','IV','V']
+playerA = np.array([5, 15, 22, 20, 25])
+playerB = np.array([25, 32, 34, 30, 27])
+x = np.arange(len(labels))
+width = 0.35
+
+fig, ax = plt.subplots()
+rects1 = ax.bar(x - width/2, np.zeros_like(playerA), width,
+                bottom=playerA, label='Player A', fill=False)
+rects2 = ax.bar(x + width/2, playerB - playerA, width,
+                bottom=playerA, label='Player B',edgecolor='black')
+#-----------------------------------------------------------------------------------------
```

Comment:
Please remove these.  A carriage return should be adequate to set the block of code off...